test_model_configs/single_population_node_with_updaters_config.txt

```diff
@@ -0,0 +1,732 @@
+                                  branch_node {
+                                    branches {
+                                      chance: 0.1930248
+                                      node{
+                                        population_node {
+                                          pools {
+                                            population_offset: 11000000000
+                                            total_population: 30478
+                                          }
+                                        }
+
+                                      }
+                                    }
+                                    branches {
+                                      chance: 0.5157664
+                                      node {
+                                        population_node {
+                                          pools {
+                                            population_offset: 11000030478
+                                            total_population: 81437
+                                          }
+                                        }
+                                      }
+                                    }
+                                    branches {
+                                      chance: 0.24120784999999997
+                                      node {
+                                        population_node {
+                                          pools {
+                                            population_offset: 11000111915
+                                            total_population: 38085
+                                          }
+                                        }
+                                      }
+                                    }
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+            }
+          }
+          updates {
+            updates{
+              conditional_merge {
+                nodes {
+                   condition {
+                      op: AND
+                      sub_filters {
+                         name: "phone_user_info.demo.gender"
+                         op: EQUAL
+                         value: "2"
+                      }
+                      sub_filters {
+                         name: "phone_user_info.demo.age_from"
+                         op: EQUAL
+                         value: "18"
+                      }
+                      sub_filters {
+                         name: "phone_user_info.demo.age_to"
+                         op: EQUAL
+                         value: "24"
+                      }
+                   }
+                   update {
+                      device_user_type: "phone"
+                      corrected_demo {
```

Comment:
Here we are setting "acting_demo". (The matrix below set corrected_demo)

Reply:
Currently the update_matrix on line 472 has the acting_demo, whereas the condition_merge has the corrected_demo. Should these be swapped?

Reply:
The conditional_merge here choose which id space to use, and copy the demo data to acting_demo accordingly.
The update_matrix below is for demo correction. It takes acting_demo as input, and sets corrected_demo.
(The updater type is irrelevant. We are talking about how the event attributes are updated and used during the labeling.)